.vscode/settings.json

```diff
@@ -19,4 +19,7 @@
 	"[typescript]": {
 		"editor.defaultFormatter": "vscode.typescript-language-features"
 	},
+	"editor.codeActionsOnSave": {
```

Comment:
Mistakenly included?

Reply:
Intentionally included. I do this automatically and most imports are already sorted, @lramos15 any concerns?